src/components/Stepper/Controls/index.tsx

```diff
@@ -45,6 +45,7 @@ const Controls = ({ buttonLabels, currentStep, disabled, firstPage, lastPage, on
           style={secondButtonStyle}
           type="submit"
           variant="contained"
+          testId={`${next.toLowerCase()}-btn`}
```

Comment:
why changing the id? can't you use text inside the button?

Reply:
It didn't had an Id before (the data-testid was empty) so I put it that way to make it consistent with every button.
If the that ID is an issue I can go with the button[type="submit"] that seems to be in every  "Submit", "Review", "Start", etc. button that submits a form

Reply:
The problem I see with that is that the test will depend on the value of `next`

Also I’m not sure if we’re still using this stepper components or we fully moved to the one from `safe-react-components`

@nicosampler do you know?

Reply:
in safe-creation, we are using the stepper from safe-react-components, but I think it is the only place where we are using it.

Reply:
The value of "next" is always the name of the button, so every button id will be "submit-btn", "review-btn", "start-btn". 
I'll try with the button[type="submit"] in my tests, since there is always 1 button displayed at the time it should work.